frontend/src/store/modules/shoots.js

```diff
@@ -455,8 +455,9 @@ const shoots = (state) => {
 }
 
 const setSortedItems = (state, rootState) => {
-  const sortBy = get(state, 'sortParams.sortBy')
-  const descending = get(state, 'sortParams.descending', false) ? 'desc' : 'asc'
+  const sortBy = head(get(state, 'sortParams.sortBy'))
+  const sortDesc = get(state, 'sortParams.sortDesc', [false])
+  const descending = head(sortDesc) ? 'desc' : 'asc'
```

Comment:
I know it has nothing to do with this PR. But it is completely confusion that a variable with the name `descending` could have a value `'asc'`? We should rename `descending` into `order` or `sortOrder`.

Reply:
✅